src/views/Records/Records.vue

```diff
@@ -58,6 +58,7 @@
           />
         </v-col>
       </v-row>
+      <Footer class="mb-2" />
```

Comment:
Im unsure why this is here and not in App.vue like discussed in a previous PR.

Reply:
it is fixed in update branch 4.1